bot/exts/moderation/infraction/infractions.py

```diff
@@ -151,6 +153,11 @@ async def send(*args, **kwargs) -> None:
         ctx.send = send
         await infr_manage_cog.infraction_append(ctx, infraction, None, reason=f"[Clean log]({log_url})")
 
+    @command()
+    async def compban(self, ctx: Context, user: UnambiguousMemberOrUser) -> None:
+        """Same as cleanban, but specifically for four days and with the ban reason used for compromised accounts."""
```

Comment:
Tested, works well. Thank you!
```suggestion
        """Same as cleanban, but specifically for the ban reason and duration used for compromised accounts."""
```

Reply:
changed in https://github.com/python-discord/bot/pull/2250/commits/a7234a824f3d4b2d052357d73d23a6e0c5cbb1b6